_maps/configs/syndicate_gorlex_komodo.json

```diff
@@ -36,20 +36,24 @@
 			"slots": 1
 		},
 		"Medic": {
-			"outfit": "/datum/outfit/job/syndicate/doctor/gorlex",
+			"outfit": "/datum/outfit/job/syndicate/paramedic/cybersun",
 			"slots": 1
 		},
 		"Mechanic": {
-			"outfit": "/datum/outfit/job/syndicate/engineer/gorlex",
+			"outfit": "/datum/outfit/job/syndicate/engineer/gec",
 			"slots": 1
 		},
 		"Trooper": {
-			"outfit": "/datum/outfit/job/syndicate/security/gorlex",
+			"outfit": "/datum/outfit/job/syndicate/security/pajamas",
```

Comment:
okay as funny as this is you should probably just put in a generic syndicate combat suit or something

they're encouraged to select their faction since it gives them armor so i don't think you also need to give them a meme set

Reply:
having them spawn with pajamas vexes me and i don't think is something that is necessary to make them pick a loadout

Reply:
nvm lol